test/support/codeflare.go

```diff
@@ -30,8 +33,16 @@ const (
 	CodeFlareTestPyTorchImage = "CODEFLARE_TEST_PYTORCH_IMAGE"
 
 	// The testing output directory, to write output files into.
-
 	CodeFlareTestOutputDir = "CODEFLARE_TEST_OUTPUT_DIR"
+
+	// The name of a secret containing InstaScale OCM token.
+	InstaScaleOcmSecretName = "INSTASCALE_OCM_SECRET_NAME"
+	// The namespace where a secret containing InstaScale OCM token is stored.
+	InstaScaleOcmSecretNamespace = "INSTASCALE_OCM_SECRET_NAMESPACE"
+	// Cluster ID for OSD cluster used in tests, used for testing InstaScale
+	OsdClusterID = "CLUSTERID"
+	// Determine if test is being run on an OSD cluster, used for testing InstaScale.
+	IsOSD = "IS_OSD"
```

Comment:
Could it be assumed that it's an OSD cluster if an OSD cluster ID has been provided?

Reply:
Yes I think so, I have updated to reflect this.